extensions/singularityapp/src/quickaddtask.tsx

```diff
@@ -0,0 +1,33 @@
+import { Toast, showToast } from "@raycast/api";
+import { exec } from "child_process";
+
+interface Task {
+    title: string;
+    description?: string;
+}
+
+async function sendTaskToServer(task: Task) {
+    return new Promise((resolve, reject) => {
+        // Команда для отправки задачи через ssh, используя sshpass для автоматической передачи пароля
+        const sshCommand = `ssh -p 2022 retrobanner@gkogan.ddns.net "echo '${task.description}' | mail -s '${task.title}' tasksja5bpu#ru@singularity-app.com"`;
+
+        exec(sshCommand, (error, stdout, stderr) => {
+            if (error) {
+                reject(`Ошибка выполнения SSH команды: ${stderr}`);
+            } else {
+                resolve("Задача отправлена");
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension. If the locale might affect functionality (e.g. using the correct unit of measurement), please use the preferences API.